fe1-web/src/core/network/validation/Validator.ts

```diff
@@ -122,3 +124,12 @@ export function validateScannableMainPublicKey(obj: any): ValidationResult {
 export function validateScannablePopToken(obj: any): ValidationResult {
   return validate(`${schemaPrefix}/qrcode/pop_token.json`, obj);
 }
+
+export function validateFederationExchange(obj: any): ValidationResult {
+  // TODO change URL before merging to master
+  // also in federation_exchange.json
+  return validate(
+    `https://raw.githubusercontent.com/dedis/popstellar/work-fe1-ljankoschek-LinkedOrgUI/protocol/qrcode/federation_exchange.json`,
```

Comment:
This can be changed to "https://raw.githubusercontent.com/dedis/popstellar/master/protocol/qrcode/federation_exchange.json" before merging as it's more of an identifier.